client/me/billing-history/controller.js

```diff
@@ -37,7 +37,7 @@ export default {
 
 			renderWithReduxStore(
```

Comment:
We render twice into `primary` element, when there is a transaction id provided.